### DISCUSSION TITLE
It is possible to use it directly on a React App without a backend?

### DISCUSSION THREAD

**Discussion by GuillemGarciaDev** [raweceek-temeletry/f1-2021-udp#49] [Q&A] [answered]
First of all, congrats mate for this awesome job! 

I want to develop a telemetry monitor with React and I don't know if I can use your library directly on a client. Is it possible? or I need to build a backend?

Thank you!

**Comment by PrimeGoose** [comment, marked as answer]
no, the browser does not support datagram which is required to get the data. you can use electron with react app, it will do the job.

**Comment by GuillemGarciaDev** [comment]
That's what I thought, thank you anyways 😄 @PrimeGoose
